test/e2e/manifests/mixins/argo-workflows-agent-ca-certificates.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: argo-workflows-agent-ca-certificates
+stringData:
+  ca-certificates: >
+    -----BEGIN CERTIFICATE-----
```

Comment:
can you add a comment for how to generate this?

Reply:
done @alexec

Reply:
cool - did you run it with invalid cert? to make sure at least one test fails?

Reply:
@alexec On mounting at `/etc/ssl/cert.pem` the pod creation fails with the following error: 
`Error: failed to create containerd task: failed to create shim: OCI runtime create failed: container_linux.go:380: starting container process caused: process_linux.go:545: container init caused: rootfs_linux.go:75: mounting "/var/lib/kubelet/pods/4cf0518f-35a4-4c01-b442-6f03bd8d3bb6/volumes/kubernetes.io~secret/argo-workflows-agent-ca-certificates" to rootfs at "/etc/ssl/cert.pem" caused: mount through procfd: not a directory: unknown`

Same for `/etc/ssl/ca-certificates.crt`

Is there a limitation on mounts?

Reply:
Okay so kubernetes does not allow us to mount secrets as a file. We have to mount the secret as a directory and I guess the key will be the filename and the value will be the content of the file in that directory.

Reply:
@alexec the workflow does give an error `x509: certificate signed by unknown authority` on providing an invalid certificate. I guess we can merge now.